### PR TITLE
style: guide link fix and detailed description

### PR DESCRIPTION
**Pull request by qzdark**
# PR Details

## Description

Due to updates in the guide, the link broke. I also added a banner to make it look similar to the guide's readme. Additionally, changed link inclusion and revised text for better clarity, aligning it with the descriptive style used in the guide's descriptions.

## Types of changes

- [ ] Dependency change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly (index.d.ts).

### CODE REVIEW

**Review by alechkos — COMMENTED** [review]
@qzdark 

### Is it possible to change the color of the `JS` part in the logo? Like maybe to gray, because for users that use light theme in GitHub the logo is displayed like this:

File: README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
-<div align="center">
+<div align="center" style="background:#0d1117; border-radius:15px">
```
[the remark this answers is not in the document]
Comment (qzdark) [reply README.md]: I've noticed it before, I just forgot about it. However, this change should look good in both dark mode and light mode

**Review by alechkos — APPROVED** [review]
good af

**Comment by qzdark** [comment]
The banners will receive another update in the future to make them look even better and cleaner. However, the link should remain the same, so everything is fine.

**Review by aliyss — APPROVED** [review]
LGTM

**Pull request merged**